time_tracker_api/api.py

```diff
@@ -2,6 +2,7 @@
     CosmosResourceExistsError,
     CosmosResourceNotFoundError,
     CosmosHttpResponseError,
+    CosmosClientTimeoutError,
```

Comment:
I have a question,  How does this help?

Reply:
I thought of handling it as 'An operation failed to complete within the specified timeout.' However, it is not necessary for a 204 message. Thanks for asking.